partiql-lang/src/test/kotlin/org/partiql/lang/compiler/IntegrationTests.kt

```diff
@@ -30,15 +32,24 @@ class TestContext {
         assertEquals(expectedIon, result.toIonValue(ION))
     }
 
+    // Executes query on async evaluator
```

Comment:
Tests in partiql-lang that tested the synchronous physical plan evaluator will also test the async physical plan evaluator. I tried to limit the amount of copied code, where possible, while also making it easy to remove the synchronous tests when we remove the synchronous evaluator in an upcoming major version.